test/unit/api/config.spec.js

```diff
@@ -81,6 +81,13 @@ describe('Config file', () => {
       expect(config.auth.htpasswd.file).toBe('./htpasswd');
       checkDefaultConfPackages(config);
     });
+
+    test('parse full.json', () => {
+      const config = new Config(parseConfigFile(resolveConf('full').replace(/.yaml$/, '.json')));
```

Comment:
Since `yaml` is the official config, the other blocks are ok, but this one, would need a special treatment, a mock file within the `partials`. 

Something else to update in this ticket needs also updates documentation https://github.com/verdaccio/website/blob/master/docs/cli.md defining the JSON format is also allowed.

Reply:
@juanpicado I'm not sure that I am understand to your last comment about mock file within the partials - could you please write one "example" test, then I can write the rest.

Reply:
If you see in this location, there is a folder called `yaml`, it is where I started to host all examples used for unit testing, I think we need one called `json` and restore the unit test your have removed. For instance:
```
const parseJSONConfigurationFile = (name) => {
    return path.join(__dirname, `../partials/config/json/${name}.json`);
  };

test('parse valid example.json', () => {
      const config = new Config(parseConfigurationFile('example1'));
      checkDefaultUplink(config);
      expect(config.storage).toBe('./storage');
      checkDefaultConfPackages(config);
    });

test('parse invalid example.json', () => {
      expect(function ( ) {
           const config = new Config(parseConfigurationFile('example1'));
       }).toThrow(Error('should fail '));
    });

```

I just made that without run it on my terminal :-) , but that's the idea, to cover those scenarios. It will help if someone changes something in your code and stop to work as is intended.

https://github.com/verdaccio/verdaccio/tree/master/test/unit/partials/config